pages/api/feedback.js

```diff
@@ -0,0 +1,41 @@
+// feedback.js api
+import mysql from 'mysql2/promise';
+
+async function connectToDatabase() {
+  return await mysql.createPool({
+    host: process.env.DB_HOST,
+    port: process.env.DB_PORT,
+    user: process.env.DB_USER,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_DATABASE
+  });
+}
```

Comment:
Instead of going this route, we can use `prisma`. Prisma already has a connection to the database and is aware of our schema, as long as you added it the the schemas file. Take a look at Nasa's landing page api code for a full example, but here's a snippet here.

`await prisma.landing_Page.create(create);`